statistics/selectivity.go

```diff
@@ -283,6 +283,66 @@ func (coll *HistColl) Selectivity(ctx sessionctx.Context, exprs []expression.Exp
 			ret *= selectionFactor
 		}
 	}
+
+	// Now we try to cover those still not covered DNF conditions using independence assumption.
+	if mask > 0 {
+		// Firstly collect single column index information.
+		colID2SingleColIdxID := make(map[int64]int64)
+		colID2SingleColIdxLen := make(map[int64]int)
+		for idxID, cols := range coll.Idx2ColumnIDs {
+			if len(cols) == 1 {
+				colID2SingleColIdxID[cols[0]] = idxID
+				colID2SingleColIdxLen[cols[0]] = coll.Indices[idxID].Info.Columns[0].Length
+			}
+		}
+		for i := range remainedExprs {
+			if mask&(1<<uint64(i)) > 0 {
```

Comment:
`if mask & (1<<uint64(i)) == 0 { continue }` is more go-style.

Reply:
Fixed that. Thanks.